### PR TITLE
Adding page for North Alabama OSS

### PR DESCRIPTION
**Pull request by blmeadows**

### CODE REVIEW

**Comment by iamvery** [comment]
This looks awesome! :tada: 

I pushed a couple changes. I thought the header was a little overbearing for the page, so I simplified it. Here's a screenshot. What do you think? :shipit:?

screen shot 2016-03-17 at 10 51 10 am

File: css/style.scss [hunk 1]
```diff
@@ -605,13 +605,10 @@
...
       img {
         display: block;
-        max-height: 99%;
+        max-height: 100%;
         position: absolute;
         top: 0; bottom:0; left: 0; right:0;
         margin: auto;
-        //min-height: 100%;
-        //min-width: 100%;
-        //position: absolute;
```
Comment (iamvery) [inline css/style.scss]: Nice! Thanks for fixing this. I was just trusting you knew what was going on, because I didn't :laughing:

**Comment by iamvery** [comment]
I just deleted some of the images that were unused.

**Comment by blmeadows** [comment]
I think it looks great! Do you think the grey background for the images is fine, or should I try to change it to white?

**Comment by iamvery** [comment]
Honestly I didn't even notice. I'm fine to ship as is and we can revisit. I think we're a little constrained by the limits of the layout. Honestly I'd love eventually to make things not so __big__, but for now it works! Shall we ship?

File: css/style.scss (hunk 1 above)
Comment (blmeadows) [inline css/style.scss]: :laughing: Yeah, that's what I get for trying to push code with the kiddos around.

**Comment by blmeadows** [comment]
I agree. Ship it! :shipit:

File: css/style.scss (hunk 1 above)
Comment (iamvery) [inline css/style.scss]: :loud_sound:

**Pull request merged**